modin/logging/config.py

```diff
@@ -112,15 +114,15 @@ def configure_logging():
 
     if LogMode.get() == "enable_api_only":
         logger.setLevel(logging.INFO)
-    logger.setLevel(logging.DEBUG)
+    else:
+        logger.setLevel(logging.DEBUG)
 
     logger.info(f"OS Version: {platform.platform()}")
     logger.info(f"Python Version: {platform.python_version()}")
-    modin_version = pkg_resources.get_distribution("modin").version
     pandas_version = pkg_resources.get_distribution("pandas").version
     num_physical_cores = str(psutil.cpu_count(logical=False))
     num_total_cores = str(psutil.cpu_count(logical=True))
-    logger.info(f"Modin Version: {modin_version}")
+    logger.info(f"Modin Version: {modin.__version__}")
```

Comment:
Why did we change the code to do this?

Reply:
The reason for this change is to still be able to use `modin` without installing it first (as I usually don't `pip install` it during development).

Reply:
Another reason I've just discovered today - `pkg_resources` isn't always present, as it's not a part of standard library but a piece of `setuptools`, which adds to our list of requirements for the _user_'s environment to run Modin, so I'm replacing those with simple `modin.__version__` and `pandas.__version__`.